_pytest/python.py

```diff
@@ -1849,12 +1849,10 @@ def toterminal(self, tw):
         for tbline in self.tblines:
             tw.line(tbline.rstrip())
         lines = self.errorstring.split("\n")
-        for line in lines:
-            if line == lines[0]:
-                prefix = 'E       '
-            else:
-                prefix = '        '
-            tw.line(prefix + line.strip(), red=True)
+        if lines:
+            tw.line('E       ' + lines[0].strip(), red=True)
```

Comment:
Just noticed that we have this in `py._code.code.py`:

```python
class FormattedExcinfo(object):
    """ presenting information about failing Functions and Generators. """
    # for traceback entries
    flow_marker = ">"
    fail_marker = "E"
```

Could please change the code use those constants instead?